tests/phpunit/tests/upload.php

```diff
@@ -77,6 +77,19 @@ public function test_upload_dir_no_yearnum() {
 		$this->assertFalse( $info['error'] );
 	}
 
+	public function test_upload_dir_yearnum() {
+		update_option( 'uploads_use_yearmonth_folders', 'year' );
```

Comment:
As above, I think it would be more consistent if the option value for the new structure was '2'.

Reply:
Ok, I can see that for all these cases.